examples/access-plugin-minimal/go.mod

```diff
@@ -3,40 +3,36 @@ module teleport-sheets
 go 1.19
 
 require (
-	github.com/gravitational/teleport/api v0.0.0-20230317154531-934cde5cb86b
-	github.com/gravitational/trace v1.2.1
+	github.com/gravitational/teleport/api v0.0.0-20231017180506-30211ac889e5
```

Comment:
I don't have a preference one way or another.  This would match how the example was previously declared, and allows our release branches to be more uniform.  It sounded like we don't really version `examples` anyway, but I don't have a concern if we want to update these.